.github/scripts/render_plots.py

```
@@ -0,0 +1,120 @@
+        current_path = f"{path}/{obj_name}" if path else obj_name
+
+        if isinstance(obj, ROOT.TDirectory):
+            rendered.extend(render_directory(obj, output_dir, current_path, max_depth - 1))
+            continue
+
+        safe_name = current_path.replace("/", "_")
+        png_path = output_dir / f"{safe_name}.png"
+
+        try:
+            meta = render_object(obj, png_path)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)